src/components/manim-youtube-carousel.js

```diff
@@ -78,15 +90,16 @@ const ManimYoutubeCarousel = () => {
                             id={info.id}
                             play={playVideos[i]}
                             onPauseCB={resumeCarousel}
-                            startAt={info.start && info.start} />
-                    )
+                            startAt={info.start && info.start}
+                        />
+                    );
                 })}
             </Carousel>
-            <button className="more-button">
-                More Made With Manim
-            </button>
+            <Link to="/made-with-manim">
```

Comment:
isn't it called `/awesome`?

Reply:
I renamed it to made-with-manim after the section on the home page. I'm assuming a lot of people are not familiar with the concept of awesome lists, so "Made with Manim" should be better, wdyt?

Reply:
> I renamed it to made-with-manim after the section on the home page. I'm assuming a lot of people are not familiar with the concept of awesome lists, so "Made with Manim" should be better, wdyt?


Actually, people don't need to be familiar with the concepts of awesome lists, since it's just the URL. I prefer `/awesome` as it's small than `made-with-manim`. Also, manim is already in the domain name so from `https://manim.community/awesome` people can understand that it contains awesome stuff made with manim. Though I would like to hear what others think.

Reply:
> I prefer `/awesome` as it's small than `made-with-manim`

On a second thought, I agree. I renamed it back to `/awesome`, though the title is still Made with Manim.